domains/game/GameStatus.jsx

```diff
@@ -1,21 +1,59 @@
-import React from "react";
-import { Text, View } from "react-native";
+import React, { useMemo } from "react";
+import { Text, View, StyleSheet } from "react-native";
 import PropTypes from "prop-types";
 
 const GameStatus = ({ game, playerName }) => {
+    const currentPlayerColor = useMemo(() => {
+        return game.players.find((player) => player.current).color;
+    }, [game.players]);
+    const getColorStyle = () => {
```

Comment:
as this function do not depends from any pros, you can move it outside of the render function, to avoid recreating it on every render